.github/travis/common.sh

```diff
@@ -51,6 +51,32 @@ if [ -z "$DATESTR" ]; then
 	fi
 fi
 
+function make_target() {
+  target=$1
+
+  if [ ! -v MAKE_JOBS ]; then
+    export MAKE_JOBS=$(nproc)
```

Comment:
I think you might want to force this to 2 on Travis?

Reply:
I detects the correct number. Not sure why we'd need to force it at the moment.